python/pyspark/sql/streaming.py

```diff
@@ -224,6 +226,494 @@ def _to_java_trigger(self, sqlContext):
             self.interval)
 
 
+class DataStreamReader(OptionUtils):
```

Comment:
then probably we have to create directory streaming and move all the code in streaming.py into separate files. I wanted to minimize the changes. we can do the bookkeeping later. what do you think?